fire.js

```
@@ -1,146 +1,97 @@
+    var my = Math.floor(mouseY);
+    for (var dx = -2; dx <= 2; dx++) {
+        for (var dy = -2; dy <= 2; dy++) {
+            var px = mx + dx;
+            var py = my + dy;
+            if(px > 0 && px < width && py > 0 && py < height) {
+                Buffer1[px][py] = 255;
+            }
+        }
+    }
+
+    for (var x = 1; x < (width - 1); x+=1){
```

Biome (1.9.4)

[error] Shouldn't redeclare 'x'. Consider to delete it or rename it.

'x' is defined here:


(lint/suspicious/noRedeclare)